psake.psm1

```diff
@@ -621,7 +621,23 @@ function ConfigureBuildEnvironment {
     $frameworkDirs = @()
     if ($buildToolsVersions -ne $null) {
         foreach($ver in $buildToolsVersions) {
-            if (Test-Path "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\$ver") {
+            if ($ver -eq "15.0") {
+                $vs2017paths = @(
+                    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise",

```

Comment:
Perhaps we should use the env var `${env:ProgramFiles(x86)}\Microsoft Visual Studio\2017\Enterprise` instead of hard-coding the path to the Program Files (x86) dir?  Not having used a 32-bit OS in a while, I can't remember if that env var is defined on a 32 bit OS.